app/code/Magento/Catalog/Ui/DataProvider/CatalogEavValidationRules.php

```diff
@@ -33,14 +33,17 @@ public function build(ProductAttributeInterface $attribute, array $data)
             $rules['validate-zero-or-greater'] = true;
         }
 
-        $validationClasses = explode(' ', $attribute->getFrontendClass());
+        $validationClasses = $attribute->getFrontendClass()
+            ? explode(' ', $attribute->getFrontendClass()) : [];
 
         foreach ($validationClasses as $class) {
             if (preg_match('/^maximum-length-(\d+)$/', $class, $matches)) {
+                // phpcs:ignore Magento2.Performance.ForeachArrayMerge
```

Comment:
Could it be fixed ?
https://github.com/magento/magento-coding-standard/blob/develop/Magento2/Sniffs/Performance/ForeachArrayMergeSniff.md

Reply:
@xmav hello! Thanks for your review. I think we can remove array_merge from this code. Please, check my changes.
Thanks!